src/wallet/bdb.cpp

```diff
@@ -285,8 +284,7 @@ bool BerkeleyDatabase::Verify(bilingual_str& errorStr)
 
     if (fs::exists(file_path))
     {
-        LOCK(cs_db);
```

Comment:
Why was this removed?

Reply:
It was there to guard mapFileUseCount which we've now removed.